parsel/selector.py

```diff
@@ -1,9 +1,9 @@
 """
-XPath selectors based on lxml
+Jmespath/XPath selectors based on lxml and jmespath
 """
-
+import json
 import sys
-
+import jmespath
 import six
```

Comment:
:lipstick: `sys` and `six` were separated by an empty line because they are built-in and third-party libraries respectively.

```
import json
import sys

import jmespath
import six
```